internal/models/v2models/chart_release_version.go

```diff
@@ -214,6 +216,10 @@ func (chartReleaseVersion *ChartReleaseVersion) validate() error {
 		return fmt.Errorf("a %T must have a terra-helmfile git reference", chartReleaseVersion)
 	}
 
+	if chartReleaseVersion.FirecloudDevelopRef == nil || *chartReleaseVersion.FirecloudDevelopRef == "" {
+		return fmt.Errorf("a %T must have a firecloud-develop git reference", chartReleaseVersion)
+	}
```

Comment:
I'm not sure this quite makes sense--every single chart release _must_ have a firecloud-develop ref?